src/main/steps/index.utils.ts

```diff
@@ -0,0 +1,5 @@
+import { CaseWithId } from '../app/case/case';
+
+export const hasSubmittedAos = (userCase: CaseWithId): boolean => {
```

Comment:
do we need a whole new utils file for just a boolean? it's only used in one place for now right? so we could put this function in the index file

Reply:
Moved it to index